server/controllers/authController.api.js

```diff
@@ -25,11 +30,17 @@ exports.login = (req, res) => {
         if (err) {
             return res.status(500).json({ message: 'an error occurred' });
         } else if (user && user.comparePassword(password)) {
-            const { name, email } = user;
-            const token = encodeToken({ name, email });
+            const { name, email, _id } = user;
+            const token = encodeToken({ name, email, _id });
+            res.cookie('t', token, { expire: new Date() + 9999 });
```

Comment:
Are using cookies or localstorage?

Reply:
I'll remove this since we've agreed to just use localstorage for now